### PR TITLE
fix: Rendering of field type and showing only required fields

### LINKED ISSUE #3406: Attendee List: Only render required fields and some of the fields are not rendering.
Since we are rendering all fields in `attendee list` due to this many incomplete fields are getting shown.
Also, some of the select fields are not getting rendered fix them.
image

### PR DESCRIPTION
**Pull request by kushthedude**
Fixes #3406

### CODE REVIEW

File: app/templates/components/event-invoice/event-info.hbs [hunk 1]
```diff
@@ -17,7 +17,7 @@
     <strong>{{t 'To'}}:</strong> {{header-date event.endsAt}}
     {{#if event.ownerName}}
       <br>
-      <strong>{{t 'Organized By'}}:</strong> {{event.ownerName}}
+      <strong>{{t 'Owned By'}}:</strong> {{event.ownerName}}
```
Comment (CosmicCoder96) [inline app/templates/components/event-invoice/event-info.hbs]: No one says owner by, you organize an event, not own it.

**Comment by kushthedude** [comment]
@CosmicCoder96 Please REview

**Comment by kushthedude** [comment]
@CosmicCoder96 Please REview

**Comment by kushthedude** [comment]
@mrsaicharan1 @shreyanshdwivedi @uds5501 Please Review

**Comment by kushthedude** [comment]
@mrsaicharan1 @shreyanshdwivedi @uds5501 Please Review

**Comment by kushthedude** [comment]
@CosmicCoder96 Please merge

**Comment by kushthedude** [comment]
image
Only required or filled up fields will get rendered

**Comment by kushthedude** [comment]
@mrsaicharan1 @shreyanshdwivedi @uds5501 @CosmicCoder96 Please Review

**Comment by kushthedude** [comment]
@mrsaicharan1 @shreyanshdwivedi @uds5501 @CosmicCoder96 Please Review

**Comment by kushthedude** [comment]
@mrsaicharan1 @shreyanshdwivedi @uds5501 @CosmicCoder96 Please Review

**Comment by kushthedude** [comment]
@CosmicCoder96 @uds5501 @shreyanshdwivedi @mrsaicharan1 Please Review

**Comment by kushthedude** [comment]
@CosmicCoder96 Please REview

**Comment by kushthedude** [comment]
@CosmicCoder96 Please REview

**Comment by CosmicCoder96** [comment]
Add a descriptionn in the PR, I am not sure what you are trying to do.

**Comment by kushthedude** [comment]
> Also, showing only required fields is not the solution? What if I have an optional field, why does it exist, if it won't be visible?

What about showing the fields only which have been filled by user @CosmicCoder96 ?

**Comment by CosmicCoder96** [comment]
@kushthedude yes but it's a form, how does it matter what he fills and what he doesn't? I didn't get your point.

**Comment by kushthedude** [comment]
> what

@CosmicCoder96 Its when the user doesn't fill some not required fields, in the summary, there are labels for the fields but the fields are empty, I am just adding a condition to show only those fields in the summary which are filled by the attendee. And also some `Select Type` fields are not getting rendered also fixed it.

**Pull request closed without merging**